screenshot.js

```diff
@@ -340,7 +338,7 @@ const _getType = id => {
           } else {
             return null;
           }
-        } else if (['png', 'jpg', 'jpeg', 'gif'].includes(ext)) {
+        } else if (['png', 'jpg', 'jpeg', 'gif', 'image'].includes(ext)) {
```

Comment:
Is image a real extension? What is the purpose of this?

Reply:
As per discussion changed the ext to appType.